src/PactNet/Verifier/Messaging/MessageScenariosFactory.cs

```diff
@@ -5,13 +5,13 @@ namespace PactNet.Verifier.Messaging
     /// <summary>
     /// Message scenarios
     /// </summary>
-    public class MessageScenarios : IMessageScenarios
+    public class MessageScenariosFactory : IMessageScenariosFactory
```

Comment:
This should be `internal` so that plugins/extensions are forced to use the abstraction

Reply:
absolutely, that's the idea. I'll adjust the accessibility